whutbbcode.user.js

```diff
@@ -430,12 +438,16 @@
 			[':test', /^$|^localhost$/],
 			['red', /(?:redacted)\.ch/],
 			['notwhatcd', /(?:notwhat)\.cd/],
+			['btn', /(?:broadcasthe)\.net/],
 			['gazellegames', /(?:gazellegames)\.net/],
+			['myanonamouse', /(?:myanonamouse)\.net/],
 			['orpheus', /(?:orpheus)\.network/],
 			['bs', /(?:bs\.lunartype)\.com/],
 			['waffles', /(?:waffles\.ch)/],
 			['indietorrents', /(?:indietorrents\.com)/],
+			['torrentday', /(?:torrentday\.com)/],
 			['oppaiti', /(?:oppaiti\.me)/]
+			['ptp', /(?:passthepopcorn\.me)/]
```

Comment:
Hey, thanks for the pull request. I appreciate it.

You should test the code changes on your local machine. Your changes break the script.

You left out commas here. The last one isn't required, but you can add it.

```suggestion
			['oppaiti', /(?:oppaiti\.me)/],
			['ptp', /(?:passthepopcorn\.me)/],
```

Reply:
Whoops, sorry about that! The build script didn't throw an error for that so I missed the broken array.

thanks for checking out the PR

Reply:
The build task doesn't actually build the script. It builds everything else. However, I should add a task in grunt to do check syntax errors just in case.